packages/story-editor/src/app/story/utils/isEmptyStory.js

```diff
@@ -40,6 +40,11 @@ const isEmptyStory = (pages) => {
     return false;
   }
 
+  const hasNoAttachment = pages[0].pageAttachment?.length === 0;
```

Comment:
Nit: we usually prefer "positive" variable names for readability, e.g.
`const hasPageAttachment` and then just check for `if(hasPageAttachment)`.

The page attachment is actually an object that consists of different props, e.g. if we remove a Page Attachment from a Page, it will still stay as a prop but empty:
```
"pageAttachment": {
        "url": "",
        "icon": "",
        "needsProxy": false
      }
```

It can also be `null` if not set.

We should probably check here for the length of the `url` instead.

Reply:
➕  for the "positive" variable names (in most cases), I'm first to blame here (see `hasNoExtraElements` and `hasNoBackgroundMedia` although this one is tricky)

![image](https://user-images.githubusercontent.com/21036224/157644474-1a7444a1-8e7f-4790-b9e8-26af3f82cf81.png)